section2/exercises/decision-making.js

```diff
@@ -39,15 +39,37 @@ Questions
 
 1. In English, using technical vocabulary, describe what is happening between lines 14 and 18.
 
+// The code is setting up two conditions. one where if you go through door one there is a bear wearing a hat.
+One where if you dont go through door one, then the bear is wearing a scarf. You will choose your answer to the question
+"Do you go through #1 or #2" and the code will run the appropriate scenario based on how your answer fits the conditions set.
+Because the current variable doorChoice is equal to one on line eight, the bear will be wearing a hat.
+
 2. What variable has a new value assigned to it after the first if statement executes?
 
+bearClothing
+
 3. If you changed the variable doorChoice to equal 3, what would the bearClothing value be?
 
+"scarf" because 3 is not strictly equal (!===) to one, so it would run the else condition leading to scarf being chosen.
```

Comment:
You caught this elsewhere but be careful of your strictly not equal syntax- `!==`